inclusive-design/idrc
# feat: update consulting page layout (resolves #65)

## Description

This PR updates the layout of the consulting page as requested.

TODO:

- [x] Vertical spacing before/after headings on mobile

## Steps to test

1. Visit https://deploy-preview-88--idrc.netlify.app/consulting.

**Expected behavior:** New page layout matches expectations.

## Additional information

Subsections can now be created based on either `<h3>` or `<h4>` elements based on a setting in the parent section. This allows the customized column division requested for the consulting page.

## Related issues

Resolves #65.
